pkgs/development/python-modules/argcomplete/default.nix

```diff
@@ -0,0 +1,24 @@
+{ buildPythonPackage, fetchPypi, lib,
+  coverage, dicttoxml, flake8, pexpect, prettytable, requests_toolbelt
+}:
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  pname = "argcomplete";
+  version = "1.8.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0sslhl1klvh92c8hjsz3y3mmnpcqspcgi49g5cik2rpbfkhcsb3s";
+  };
+
+  doCheck = false; # bash-completion test fails with "compgen: command not found".
+
+  propagatedBuildInputs = [ coverage dicttoxml flake8 pexpect prettytable requests_toolbelt ];
```

Comment:
are all of these runtime dependencies? To me, it seems like some (like flake8) are only needed for tests